src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIfStatement.java

```diff
@@ -155,8 +156,28 @@ public void checkBooleanStatements() {
                 && (Integer.valueOf(d) instanceof Integer) == true) { // ok
             return;
         }
-    }
-
 
+        boolean x = false, y = true, z = false;
```

Comment:
please move all new cases(that you adding only) to new Input files, we already agreed to keep Input size not above 120 lines.
We have enormous amount of bad examples, but lets not grow it further.

Reply:
My bad, I forgot about it, as I was touching that file, I extracted some inputs to the new file I made to get under the limit and removed a suppression.